### PR TITLE
RSClient: WIP refactoring

### PR DESCRIPTION
**Pull request by Lucwousin**
only reason it's wip cause I may or may not like my names enough ree

### CODE REVIEW

**Comment by Lucwousin** [comment]
ty to https://github.com/shaheenkdr/ExoDemo/blob/master/library/src/main/java/com/google/android/exoplayer2/extractor/ogg/VorbisUtil.java for making the music kinda doable

File: runescape-client/src/main/java/Sprite.java [hunk 1]
```diff
@@ -713,7 +713,7 @@ public void method314(int var1, int var2, int var3, int var4, int var5, int var6
 
    @ObfuscatedName("ay")
    @Export("drawRotatedMaskedCenteredAround")
-   public void drawRotatedMaskedCenteredAround(int x, int y, int maskWidth, int maskHeight, int spriteWidth, int spriteHeight, int rotation, int var8, int[] maskXStarts, int[] maskXWidths) {
+   public void drawRotatedMaskedCenteredAround(int x, int y, int maskWidth, int maskHeight, int spriteWidth, int spriteHeight, int rotation, int var8, int[] maskStartXs, int[] maskXWidths) {
       try {
```
Comment (Owain94) [inline runescape-client/src/main/java/Sprite.java]: var8 -> zoom

**Comment by Owain94** [comment]
Niet in je scope maar als we toch bezig zijn

Regel 296 in Scene.java

addTile(int plane, int x, int y, int shape, int lotation, int texture, int heightSw, int heightNw, int heightNe, int heightSe, int underlaySwColor, int underlayNwColor, int underlayNeColor, int underlaySeColor, int overlaySwColor, int overlayNwColor, int overlayNeColor, int overlaySeColor, int underlayRgb, int overlayRgb)

**Comment by Lucwousin** [comment]
https://i.imgur.com/KmBfNog.png

**Comment by Lucwousin** [comment]
this has basically gotten merged by last update :S

**Pull request closed without merging**